pkgs/applications/misc/taskwarrior-tui/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "taskwarrior-tui";
+  version = "0.9.5";
+
+  src = fetchFromGitHub {
+    owner = "kdheepak";
+    repo = "taskwarrior-tui";
+    rev = "v${version}";
+    sha256 = "1348ypjphm5f46civbrcxbbahwwl2j47z1hg8ndq1cg2bh5wb8kg";
+  };
+
+  # Because there's a test that requires terminal access
```

Comment:
if this is only one test among many, please disable it with a patch or sed.

Reply:
One among four afaics, I think value of these three other tests vs. effort to disable the one is too much with too little benefit.

Reply:
`substituteInPlace src/main.rs --replace 'fn test_main' $'#[ignore]\nfn test_main'`

Reply:
I think it is fine here to disable tests.